backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala

```diff
@@ -243,6 +243,12 @@ class TestOperator extends WholeStageTransformerSuite {
     checkLengthAndPlan(df, 1)
   }
 
+  test("bin function") {
+    val df = runQueryAndCompare("SELECT bin(l_orderkey) " +
+      "from lineitem limit 1") { _ => }
+    checkLengthAndPlan(df, 1)
```

Comment:
Can we use `checkOperatorMatch` to assert a Project operator?

Reply:
Already updated, thanks!